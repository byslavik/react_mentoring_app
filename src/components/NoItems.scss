.noItems {
  font-size: 70px;
  text-align: center;
  color: #999;
  font-weight: bold;
  text-transform: uppercase;
  width: 100%;
  padding: 20px;
}