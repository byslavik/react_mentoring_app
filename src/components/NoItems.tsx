import * as React from 'react';
import * as s from './NoItems.css';

export const NoItems = () => {
  return <div className={s.noItems}>
    No Items Found
  </div>
}
