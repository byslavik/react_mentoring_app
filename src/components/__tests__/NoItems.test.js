import * as React from 'react';
import {NoItems} from '../NoItems';
import * as renderer from 'react-test-renderer';


describe('NoItems component', ()=>{
  it('NoItems component shows correctly', () => {
    const tree = renderer.create(
      <NoItems />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
})