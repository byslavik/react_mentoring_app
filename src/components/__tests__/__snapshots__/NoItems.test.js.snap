// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoItems component NoItems component shows correctly 1`] = `
<div
  className={undefined}
>
  No Items Found
</div>
`;
